tasks/main.yml

```diff
@@ -23,21 +23,88 @@
     state: present
   notify: "Restart Fluentbit."
 
-- name: "Create needed directory."
+- name: "Create group td-agent-bit"
+  group:
+    name: td-agent-bit
+    state: present
+    system: True
+
+- name: "Create user with adm group"
+  user:
+    name: td-agent-bit
+    create_home: False
+    group: td-agent-bit
+    state: present
+    groups: adm
+    system: True
+    append: True
+
+- name: Ensure unit directory is present
+  file:
+    path: /etc/systemd/system/td-agent-bit.service.d
+    state: directory
+    mode: '0755'
+
+- name: Add unit override.conf file
+  copy:
+    src: override.conf
+    dest: /etc/systemd/system/td-agent-bit.service.d/override.conf
+
+- name: Add lua file
+  copy:
+    src: functions.lua
+    dest: /etc/td-agent-bit/functions.lua
+
+- name: "Create run directory."
   file:
     path: "{{ fluentbit_dbs_path }}"
     state: directory
     mode: 0755
+    owner: td-agent-bit
+    group: td-agent-bit
   when: fluentbit_dbs_path | length > 0
 
+- name: "Create logs directory."
+  file:
+    path: "{{ fluentbit_log_path }}"
```

Comment:
what is the default of fluentbit_log_path ?

Reply:
By default there is no log path, errors are only on stderr